packages/web/src/components/Grid/README.md

```diff
@@ -0,0 +1,46 @@
+# Grid
+
+Use Grid to build multiple column layouts. This Grid works on twelve column
+system, and it contains variants of 12, 6, 4, 3, 2, and 1 column for each breakpoint.
+And additionally one centered narrow column.
+
+```html
+<div class="Grid Grid--cols-1">
```

Comment:
Ah, I'm sorry, I must have said it wrong. I meant to put it in the example, like this:

<img width="1095" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/154557749-d29f94a0-30cc-4f61-85c9-4c6ab1a7c0ee.png">